completion/available/packer.completion.bash

```diff
@@ -20,12 +20,29 @@
 # contributors for the inspiration and especially to the bash-completion
 # team in general.
 #
+# The MIT License (MIT)
```

Comment:
Packer [completes it's self](https://www.packer.io/docs/commands#autocompletion). I think it's safe to copy the pattern you have in vault

```
complete -C packer packer
```

Reply:
great idea, done